packages/plugin-api/src/model.ts

```diff
@@ -0,0 +1,25 @@
+import * as fs from 'fs-extra';
+import * as path from 'path';
+
+const boa = require('@pipcook/boa');
```

Comment:
```suggestion
import * as boa from '@pipcook/boa';
```

Reply:
Does boa support this format of import now. Probably need .d.ts files for typescript